src/Jobs/MemberOrchestrator.php

```diff
@@ -123,13 +124,20 @@ private function processMappingBase()
         $nickname = null;
         $pending_drops = collect();
         $pending_adds = collect();
+        $nickname_diff = false;
 
         $discord_user = DiscordUser::where('discord_id', $this->member->user->id)->first();
 
         if (! is_null($discord_user)) {
-            if (! is_null($discord_user->group->main_character))
-                $nickname = $discord_user->group->main_character->name;
-
+            if (! is_null($discord_user->group->main_character)) {
+                $corp_id = $discord_user->group->main_character->corporation()->corporation_id;
+                $corp = CorporationInfo::where('corporation_id', $corp_id)->first();
+                $ticker = $corp->ticker;
+                $mainname = $discord_user->group->main_character->name;
+                $nickname = "[".$ticker."] ".$mainname;
+                if ($this->member->nick != $nickname)
+                    $nickname_diff = true;
+            }
```

Comment:
Can you just add a blank link before the next keyword ?
Just to avoid stuff to be packed together.

```
}

foreach(....)
```